.dependabot/config.yml

```diff
@@ -1938,5 +1937,4 @@ update_configs:
     automerged_updates:
     - match:
         dependency_name: demisto/*
-        update_type: all
-
+        update_type: all
```

Comment:
please make sure we have a new line in this file. Or just revert you changes as they are not needed.